### ISSUE TITLE
Rego http.send header and connection refused issue with the plugin

### ISSUE THREAD

**Issue by nxf5025** [open-policy-agent/opa-docker-authz#37]
Seems like there's an issue with using the http.send (https://www.openpolicyagent.org/docs/latest/policy-reference/#http) as part of this plugin.

When executing with headers I get the following:
```
response := http.send({"method": "post", "url": "http://<hostname>/imageCheck", "body": {"image": "hello-world"}, "headers": {"Content-Type": "application/json"}})

docker: Error response from daemon: plugin openpolicyagent/opa-docker-authz-v2:0.4 failed with error: AuthZPlugin.AuthZReq: /opa/policies/authz.rego:17: eval_type_error: http.send: operand 1 invalid request parameters(s): {"headers"}.
```

When I remove the headers I have the following:
```
response := http.send({"method": "post", "url": "http://<hostname>/imageCheck", "body": {"image": "hello-world"}})

docker: Error response from daemon: plugin openpolicyagent/opa-docker-authz-v2:0.4 failed with error: AuthZPlugin.AuthZReq: /opa/policies/authz.rego:18: eval_internal_error: http.send: Post http://<hostname>/imageCheck: dial tcp: lookup <hostname> on [::1]:53: read udp [::1]:42220->[::1]:53: read: connection refused.
```

I'm able to successfully send the POST request using curl from the server so it does not look to be a port issue.

**Comment by tsandall** [comment]
@nxf5025 the first issue looks like it's caused by the fact that the vendored version of OPA is old and the built-in function does not support the headers field. The latest version of OPA does so we just need to re-vendor.

That second issue is odd. It looks like the plugin can't resolve the hostname or is unable to connect to the resolved IP. Not sure which one.

**Comment by anderseknert** [comment]
This is fixed by #55.

The issue was that the plugin by default doesn't have access to the network. Additionally, the base image used did not have any certificates bundled, so next up would be certificate validation errors.. also fixed.

**Comment by anderseknert** [comment]
Looks like I forgot to close this before..
